src/serialbox-python/serialbox/core.py

```diff
@@ -75,11 +75,4 @@ def __str__(self):
     def __repr__(self):
         return "<Config {0}>".format(self.instance.compile_options)
 
-def init_serialbox():
```

Comment:
@havogt We don't need this anymore? It's not clear for me why it's removed...

Reply:
I have no idea what it did. Apparently we have a 2 ways of writing errors:
- [default is exiting directly on the error](https://github.com/eth-cscs/serialbox2/blob/89768abf2ed08069c1d5aca0ced908c0375bfb7b/src/serialbox-c/ErrorHandling.cpp)
- [the other keeps some error state](https://github.com/eth-cscs/serialbox2/blob/89768abf2ed08069c1d5aca0ced908c0375bfb7b/src/serialbox-c/ErrorHandling.cpp#L62)

The problem with the second one, which was used, is that it segfaults before you get any error. I didn't want to spend time to fix this here.

Reply:
The result of my change is that we now get an error if something fails from python.